### PR TITLE
Ember Engine for Kubernetes Secrets Engine

### PR DESCRIPTION
**Pull request by zofskeez**
The Kubernetes secrets engine will reside within an Ember Engine and this is the initial setup. The blueprint is out of date when running `ember g in-repo-engine kubernetes` so I updated some of the generated files to align with the [docs](https://ember-engines.com/docs/quickstart#configuring-your-engine).

### CODE REVIEW

File: ui/app/router.js [hunk 1]
```diff
@@ -154,6 +154,7 @@ Router.map(function () {
         this.route('backends', { path: '/' });
         this.route('backend', { path: '/:backend' }, function () {
           this.mount('kmip');
+          this.mount('kubernetes');
```
Comment (hashishaw) [inline ui/app/router.js:157]: While this is in development, should we put this within the check for config.environment like the pki engine?

File: ui/lib/kubernetes/addon/engine.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import Engine from '@ember/engine';
+
+import loadInitializers from 'ember-load-initializers';
+import Resolver from 'ember-resolver';
+
+import config from './config/environment';
+
+const { modulePrefix } = config;
+
+export default class YourEngine extends Engine {
```
Comment (hashishaw) [inline ui/lib/kubernetes/addon/engine.js:10]: Let's name this so it's specific 
```suggestion
export default class KubernetesEngine extends Engine {
```

**Review by hashishaw — APPROVED** [review]
Just a couple small cleanup items, otherwise 🚀

File: ui/lib/kubernetes/addon/engine.js (hunk 2 above)
Comment (zofskeez) [reply ui/lib/kubernetes/addon/engine.js:10]: Good catch! Darn generator and its generic naming!

**Pull request merged**